src/precice-aste-run.cpp

```diff
@@ -8,16 +9,11 @@ int main(int argc, char *argv[])
 {
   auto context = aste::initializeMPI(argc, argv);
   auto options = getOptions(argc, argv);
-
-  if (options["verbose"].as<bool>()) {
-    boost::log::core::get()->set_filter(logging::trivial::severity >= logging::trivial::debug);
-  }
+  addLogSink(options["verbose"].as<bool>());
```

Comment:
This will trigger something like this `( %Severity% >= debug ) and %ASTE%---[ASTE:A:0] ASTE Running in mapping test mode`. Might look a bit confusing, right? This is at lest the first logger message I get.

Reply:
The message is clear, but the first part `( %Severity% >= debug ) and %ASTE%` is not supposed to appear, right? What should it tell me?

Reply:
I have no idea where this part of the message is coming from. Can you check again after a rebase?

Reply:
Seems to be resolved.